clients/deltalake/core/src/test/java/org/projectnessie/deltalake/ITDeltaLog.java

```diff
@@ -44,6 +47,21 @@
 
   @TempDir File tempPath;
 
+  private NessieClient nessieClient;
```

Comment:
Why does this have to live here, why not in abstract test? (same for other tests)

Reply:
see my comment in https://github.com/projectnessie/nessie/pull/1783/files#r691250922

Reply:
yes, I don't think that answered the question. As per my [comment](https://github.com/projectnessie/nessie/pull/1783#pullrequestreview-736888401) I don't see what is wrong w forcing 1 (pre1.0) client version for all these tests and using it from the abstract test. The solution here creates a lot of duplicate code and confusing pom changes imho

Reply:
but won't we eventually end up having different nessie-client versions for Delta and Iceberg? How do we guarantee that all the things use the same `client.nessie.version` when Delta/Iceberg release at a different pace?

Reply:
my thinking is the following:

1. we only need this pre-1.0 as we will be introducing backwards compatibility soon
2. the delta code will soon either be deleted or moved to the delta repo depending on discussions w/ Databricks.

Reply:
ok seems reasonable. Pushed a new simplified version and renamed `iceberg.nessie.version` to `client.nessie.version`